core_test.go

```diff
@@ -3412,3 +3412,23 @@ func TestNewRotatedRect2f(t *testing.T) {
 	}
 
 }
+
+func TestNewMatFromPointVector(t *testing.T) {
+
+	img := IMRead("images/face.jpg", IMReadColor)
```

Comment:
Not sure why you need `img` in this test. You could just set `size` to some specific value.